.github/workflows/operator-verify.yaml

```diff
@@ -7,7 +7,12 @@ on:
     paths-ignore:
       - 'docs/**'
       - 'examples/**'
-
+    types:
```

Comment:
In default case jobs are fired on: opened, reopened, synchronize. We additionally need: ready_for_review and converted_to_draft.